integration-tests/narayana-lra/src/test/java/org/acme/quickstart/lra/LRAParticipantTestResourceLifecycle.java

```diff
@@ -0,0 +1,55 @@
+package org.acme.quickstart.lra;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import org.jboss.logging.Logger;
+import org.testcontainers.containers.GenericContainer;
+
+import io.narayana.lra.LRAConstants;
+import io.narayana.lra.client.NarayanaLRAClient;
+import io.quarkus.test.common.QuarkusTestResourceLifecycleManager;
+
+public class LRAParticipantTestResourceLifecycle implements QuarkusTestResourceLifecycleManager {
+    private static final Logger log = Logger.getLogger(LRAParticipantTestResourceLifecycle.class);
+    private static GenericContainer<?> registry;
+    private static String coordinatorEndpoint;
+
+    // the endpoint on wich an LRA coordinator is listening
+    public static String getCoordinatorEndpoint() {
+        return coordinatorEndpoint;
+    }
+
+    @Override
+    public Map<String, String> start() {
+        registry = new GenericContainer<>("jbosstm/lra-coordinator:5.12.0.Final")
```

Comment:
Maybe we can use latest here or parse the version dynamically from the properties? More a question to the quarkus team in how this is handled elsewhere.

Reply:
It's a test. And other integration tests use a similar approach.

Reply:
ok, just to remember to bump this version when the Narayana update will be done all the time seems a little overkill. Not sure if dependabot can handle such a scenario.

Reply:
When we migrate to a new version of narayana the version will need updating in various places, I'm still not seeing the benefit of making this particular change generic (if I could even figure out a clean way to do that).

Reply:
dependabot won't update it.
But, I believe it's fine.

Reply:
okay

Reply:
What is a periodic reminder?

Reply:
a periodic todo task that remind you to update the version once in a while.

Reply:
I am not sure what you are asking us to do but we try to track our users and update accordingly, for example we already do this with wildfly and quarkus is another platform we monitor,